lib/wallet/http.js

```diff
@@ -837,6 +784,501 @@ class HTTP extends Server {
       await req.wallet.resend();
       res.json(200, { success: true });
     });
+
+    // Wallet Name States
+    this.get('/wallet/:id/name', async (req, res) => {
+      const height = this.wdb.height;
+      const network = this.network;
+
+      const names = await req.wallet.getNames();
+      const items = [];
+
+      for (const ns of names)
+        items.push(ns.getJSON(height, network));
+
+      res.json(200, items);
+    });
+
+    // Wallet Name State
+    this.get('/wallet/:id/name/:name', async (req, res) => {
+      const valid = Validator.fromRequest(req);
```

Comment:
We do not need this `Validator` object anymore. The name is passed in the resource URL.

Reply:
A `Validator` is still used to parse data from the resource URL in other parts of the codebase, see https://github.com/handshake-org/hsd/blob/master/lib/node/http.js#L159

Reply:
For sure. I just noticed my error and deleted the rest of these comments.